apps/leo_gateway/test/leo_nfs_integration_tests.sh

```diff
@@ -110,31 +116,47 @@ function make_many_files() {
     return 0
 }
 
+function check_dummy() {
+    if [ ! -d $DST_OLD_DIR ]; then
+        echo "[Failed] Old Dummy Dir File Missing"
+        return 1
+    elif [ ! -d $DST_NEW_DIR ]; then
+        echo "[Failed] NEW Dummy Dir File Missing"
+        return 1
+    elif [ ! -d $DST_BOTH_DIR ]; then
+        echo "[Failed] OLD+NEW Dummy Dir File Missing"
+        return 1
+    fi
+    return 0
+}
+
 # Tests
 {
     # try block
     eval $CMD_MOUNT &&
     eval $CMD_DF &&
+    check_dummy &&
     eval $CMD_MKDIR &&
     eval $CMD_CP_1K &&
     eval $CMD_CP_1M &&
     eval $CMD_CP_50M &&
     eval $CMD_DIFF_1K &&
     eval $CMD_DIFF_1M &&
     eval $CMD_DIFF_50M &&
-    ls_validate_num_of_childs $MOUNT_DIR 4 && 
+    ls_validate_num_of_childs $MOUNT_DIR 7 && 
     eval $CMD_RM_1K &&
-    ls_validate_num_of_childs $MOUNT_DIR 3 &&
+    ls_validate_num_of_childs $MOUNT_DIR 6 &&
     eval $CMD_MV_50M &&
-    ls_validate_num_of_childs $MOUNT_DIR 2 &&
+    sleep 5 &&
```

Comment:
As I commented the related issue you filed on https://github.com/leo-project/leofs/issues/629
Please remove this line if there is no problem with N=1 or N=W=D.